public/playersBetController.js

```diff
@@ -0,0 +1,147 @@
+"use strict";
+export default class playersBetController {
+  constructor(players) {
```

Comment:
It would be best to receive a reference to the board instead of the players, otherwise the bet controller will never know when the players list has been updated, right now it works fine because Board is mutating the array of players, but when that is no longer the case, the controller will break.